SPECS/ceph/CVE-2012-6708.patch

```diff
@@ -0,0 +1,27 @@
+From c993d7d4b9ddb5d9dcb0c4120f798730aacfff0a Mon Sep 17 00:00:00 2001
+From: Kevin Lockwood <v-klockwood@microsoft.com>
+Date: Mon, 13 Jan 2025 14:53:54 -0800
+Subject: [PATCH] Fix CVE-2012-6708
+
+Source: https://github.com/jquery/jquery/commit/05531fc4080ae24070930d15ae0cea7ae056457d
```

Comment:
Suggestion: I'd use the original subject from the upstream commit, so that the reader gets more info about this change. I tend to just use GitHub patches, since they package things nicely for me: https://github.com/jquery/jquery/commit/05531fc4080ae24070930d15ae0cea7ae056457d.patch.

Reply:
Will do